### ISSUE TITLE
Secowner TP Lure not working

### ISSUE THREAD

**Issue by rbruhn** [OpenCollar/opencollar#523]
v3.997
When an individual is added as secowner the tp lure does not work. I checked the secowners exceptions and nothing is checked there.

**Comment by WendyStarfall** [comment]
Tested on a fresh default collar.

I have not been able to make the tp lure fail by adding a secondary owner. The information to reproduce is a bit sparse, so I also let this secondary owner strict-leash me and then send a tp lure elsewhere, this worked correctly as well.

In OpenCollar stock versions secondary owners don't receive any defaulted exceptions. If the wearer was locked down in travel restrictions or grabbed though the relay and grounded a secondary owner would have to be exempt first to defeat the restriction.

Hope this helps!

**Comment by rbruhn** [comment]
I will try a fresh install of scripts into the collar and report back. The collar was updated from past versions so maybe something went wrong.

**Comment by rbruhn** [comment]
I had her pick up new collar from OC Temple. She wore the collar, turned on rlv, added me as a secowner. I then tried to tp her to me and it would not work. I also used strict leashing and the tp lure would not work. If added as owner, the tp lure works.

Firestorm64 4.6.942974
OpenCollar Version 3.997
Detected Version of RLV: 2.8.0 (RLVa: 1.4.10)

I'm not sure what other information I can give you to reproduce this issue given that it was a new collar from the OC Temple itself. Nothing modified, not third party, no plugins, and even had her remove all attachments that might be interfering.
